turbinia/output_manager.py

```diff
@@ -52,19 +52,21 @@ def __init__(self):
     self.is_setup = False
 
   @staticmethod
-  def get_output_writers(name, uid, remote_only):
+  def get_output_writers(name, uid, reqid, remote_only):
```

Comment:
For consistency, please change this to `request_id`

Reply:
Also, doesn't it already add the request ID to the path?

Reply:
Had a look at both code and current output structure and I don't see where it adds request_id? Where do you see it?

Reply:
I cherry picket turbinia-ng. I can modify reqid per request_id. 
In output_manager.py is reqid and in output_manager_test.py is request_id.

Reply:
@hacktobeer I thought it adds request_id in turbiniactl https://github.com/google/turbinia/blob/75b66c6d4f4765d45ee5d7574e1189e1def598c3/turbinia/turbiniactl.py#L762, but I do see it being called with task_id as well and get your point on annoyance.

Reply:
@dianakramer we use `request_id` all over the code. Keeping it consistent will make it more readable and less confusing :)